properties.schema

```diff
@@ -301,6 +310,19 @@
                 "validators": [],
                 "help": "The tooltip text to display on hover over this item",
```

Comment:
@kirsty-hames That would make sense to me since it is also shown when focused via keyboard navigation. Probably would want to keep it consistent with other plugins and core schemas as well?

Reply:
Done